oai_agents/common/arguments.py

```diff
@@ -40,21 +40,9 @@ def get_arguments(additional_args=[]):
     parser.add_argument('args', nargs='?', type=str, default='', help='')
 
     parser.add_argument('--epoch-timesteps', type=int)
-    parser.add_argument('--total-train-timesteps', type=int)
     parser.add_argument('--n-envs', type=int, help='Number of environments to use while training')
     parser.add_argument('--teammates-len',  type=int)
-    parser.add_argument('--groups-num-in-population', type=int, default=2, help="Number of groups in the FCP population. Each group is a list of teammates_len agents")
-    parser.add_argument(
-        '--train-types', type=str, nargs='*', default=[], 
-        choices=[v for k, v in vars(TeamType).items() if not k.startswith('__')],
-        help='Specify the train types to include, e.g., H M MID L R RHM RHL RML HLR SP'
-    )
-    parser.add_argument(
-        '--eval-types', type=str, nargs='*', default=[], 
-        choices=[v for k, v in vars(TeamType).items() if not k.startswith('__')],
-        help='Specify the evaluation types to include, e.g., H M MID L R RHM RHL RML HLR SP'
-    )
-
+    
```

Comment:
I want to retain the flexibility to use different train_types and eval_types for our FCP experiments. This is important because we are not yet certain which combinations will help our agent learn the best policy. However, if you have already designed any functions that allow us to easily change these parameters, I am okay with those changes.

Reply:
I fixed this in the PR 8. when we merge PR 8 all the train and eval types will be available in arguments.py

Reply:
https://github.com/HIRO-group/multiHRI/blob/c207313cea5649a49369c4aa3385b66f96663753/oai_agents/common/arguments.py#L47